Add November security meeting minutes (#580)
spdx/meetings: security/2023-11-08.md

# SPDX Security Team Meeting - November 8, 2023
## Attendees
* Rose Judge
* Dick Brooks
* Jeff Schutt
* Maximilian Huber

## Agenda
* Approve meeting minutes PR from 10/25
  * https://github.com/spdx/meetings/pull/572
* Review security relationships
  * https://github.com/spdx/spdx-3-model/pull/548/files
  * Review Rose's PR
* Discuss CISA KEV use case

##  Notes
### Approve 10/25 meeting PR:
* https://github.com/spdx/meetings/pull/572
* Approved
  
### Review Security Relationships
* https://github.com/spdx/spdx-3-model/pull/548/files - general relationship PR that touches a few security relationships
* Rose has proposal PR for other security relationships: https://github.com/spdx/spdx-3-model/pull/550
* Need to clarify where to put additional properties on relationships that come with specific relationship types (i.e. `hasAssessmentFor`)

### CISA KEV use case
* How would this be modeled using current relationships?

> * A CVE is a KEV or not a KEV

Reviewer: ```suggestion
* A CVE is listed on a KEV or is not listed on a KEV, as described [here](https://github.com/spdx/spdx-3-model/blob/main/model/Security/Classes/ExploitCatalogVulnAssessmentRelationship.md)
```

>   * Indicates that a CVE MUST be fixed to sell product to the government because it has been observed being exploited in the wild.

Reviewer: ```suggestion
  * Could indicate that a CVE MUST be fixed, e.g. to meet security policy or to sell product to certain customers because it has been observed being exploited in the wild.
```

>   * Should this be a property or assessment relationship?

Reviewer: I don't recall having this discussion. I would note that the team decided to make this a relationship for consistency with the 3.0 model approach and the other security assessment relationships.
```suggestion
```
